src/translations/locales/common_hi-HI.json

```diff
@@ -7,7 +7,7 @@
     "loading": "लोड हो रहा है",
     "getDirections": "दिशा - निर्देश प्राप्त करें",
     "makeABooking": "बुकिंग करें",
-    "kmAway": "{{distance}}किमी दूर",
```

Comment:
I tried my best here by matching the output. I'm 90% confident it's correct.

I believe कि means kilo and मी means meters.